tools/dev_console/static/styles/tokens.css

```
@@ -0,0 +1,58 @@
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  min-height: 100vh;
+  color: var(--ink);
+  background: var(--bg);
+  font-family: "Aptos", "Segoe UI", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Aptos" (font-family-name-quotes)

(font-family-name-quotes)